metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableConnectionBuilder.java

```diff
@@ -94,6 +103,7 @@ public BigtableDataClient dataClient() {
 
         @Override
         public void close() throws Exception {
+            mutator.close();
             session.close();
         }
```

Comment:
This is the one that could return a future now. It would used in Managed to correctly serialize the shutdown (now mutator shuts down, _then_ session in a single thread).